src/components/Multiselect/Multiselect.vue

```diff
@@ -25,7 +27,7 @@
       </span>
       <span
         v-if="(placeholder && selected.length === 0) || hideChips"
```

Comment:
não tem relação com o pr mas colocaria isso numa variável externa

outra coisa, 0 é false então poderia simplesmente colocar !selected.length

se achar válido e tiver tempo:

```suggestion
       const externalVar = (placeholder && !selected.length) || hideChips;
        v-if="externalVar"
```

Reply:
@lucasalbuquerque ajuste feito!